components/x-gift-article/src/HighlightSection.jsx

```diff
@@ -0,0 +1,40 @@
+import { h } from '@financial-times/x-engine'
+import { ShareType } from './lib/constants'
+
+export default ({
+	shareType,
+	hasHighlights,
+	includeHighlightsHandler,
+	includeHighlights,
+	isGiftUrlCreated
+}) => {
+	if (shareType === ShareType.enterprise && hasHighlights) {
+		if (isGiftUrlCreated && includeHighlights) {
+			return (
+				<div className="x-gift-article__checkbox">
+					<div className="x-gift-article__highlight-shared">highlights VISIBLE TO RECIPIENTS</div>
```

Comment:
why there a difference of case here? the uppercase should be set via text-transformation